specs/swagger.yml

```diff
@@ -3890,6 +3890,7 @@ definitions:
     type: string
     enum:
       - Windows
+      - WindowsServer
```

Comment:
This line allows WindowsServer as an OS type in the API. The rest of the back end assumes the validation has properly filtered values and doesn't inspect them past string comparisons.

Updates to the go-sdk and Window Destktop Edge are needed to submit the new "WindowsServer" values in posture responses.